XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -640,7 +658,7 @@ private void dataGridViewHosts_CheckBoxClicked(object sender, EventArgs e)
         {
             foreach (PatchingHostsDataGridViewRow row in dataGridViewHosts.Rows)
             {
-                var host = row.Tag as Host ?? Helpers.GetMaster(row.Tag as Pool);
+                var host = row.Tag as Host;

```

Comment:
The original code is still needed here, because we also need to call EnableRow for the pool rows. This EnableRow is used to dissallow pre and post Ely servers to be selected at the same time (see commit 33979bb1f2053c9c932ffa52be3c0aa5146dc72a